core/module.h

```diff
@@ -287,6 +288,14 @@ class alignas(64) Module {
   virtual struct task_result RunTask(void *arg);
   virtual void ProcessBatch(bess::PacketBatch *batch);
 
+  /*
+   * If a derived Module overrides OnEvent and returns ENOTSUP for a particular
+   * Event `e` it will be invoked for each instance of the derived Module
+   * whenever `e` occours. See `event.h` for details about the various event
+   * types and their semantics/requirements when it comes to modules.
+   */
+  virtual int OnEvent(bess::Event) { return ENOTSUP; }
```

Comment:
To be consistent with other code, please consider using `-ENOTSUP`. The remaining C-style code expects int functions return negative numbers to report errors.

Reply:
Sure thing